### PR TITLE
Use ini based configuration file

### LINKED ISSUE #2: Configuration file
Use a separate file for configuration so that there is no need to edit the settings.py manually. `.ini` Files seem like a good fit, the python [configparser](https://docs.python.org/3/library/configparser.html) provides an easy interface to those.

### PR DESCRIPTION
**Pull request by em0lar**
closes #2

### CODE REVIEW

File: src/chaosinventory/settings.py [hunk 1]
```diff
@@ -1,35 +1,24 @@
...
-https://docs.djangoproject.com/en/3.1/topics/settings/
+config = configparser.RawConfigParser()
 
-For the full list of settings and their values, see
-https://docs.djangoproject.com/en/3.1/ref/settings/
-"""
+if 'CHAOSINVENTORY_CONFIG_FILE' in os.environ:
+    config.read_file(open(os.environ.get('CHAOSINVENTORY_CONFIG_FILE'), encoding='utf-8'))
+else:
+    config.read(['/etc/chaosinventory/chaosinventory.cfg', 'chaosinventory.cfg'], encoding='utf-8')
```
Comment (clerie) [inline src/chaosinventory/settings.py:10]: How do I read this order? Is the file in the same directory read first or the file from etc?

**Review by clerie — APPROVED** [review]
Looks good overall :D

File: src/chaosinventory/settings.py (hunk 1 above)
Comment (em0lar) [reply src/chaosinventory/settings.py:10]: I think the config files will be checked in the following order:
1. the file at the path `CHAOSINVENTORY_CONFIG_FILE`
2. `/etc/chaosinventory/chaosinventory.cfg`
3. `chaosinventory.cfg`

**Review by e1mo — APPROVED** [review]

File: src/chaosinventory/settings.py (hunk 1 above)
Comment (em0lar) [reply src/chaosinventory/settings.py]: 2. and 3. were swapped as discussed on 2020-01-29

**Review by adnidor — APPROVED** [review]

**Pull request merged**